src/templates/category.js

```diff
@@ -150,6 +155,17 @@ export default function Category({ data }) {
                   labelText="Gimme everything!"
                   icon={faBook}
                 />
+                <div className="rule-category-checkbox">
+                  <label className="rule-category-checkbox-label">
+                    <input
```

Comment:
lets make it red!

Reply:
Done - the checkbox and badge are in ssw-red